practices/automateRepeatableTasks/readme.md

```diff
@@ -0,0 +1,65 @@
+# Automate Repeatable Tasks
+
+Humans are error prone; we are very bad at doing the same thing over and over. Machines, on the other hand, are very good at repeatable tasks! When working on a software project, we should strive to reduce any manual steps as much as possible. This will save time, reduce errors, but also add predictability to our software. "Works on my machine" is never an acceptable excuse! =)
```

Comment:
The voice on this segment feels inconsistent with the rest of the document

Reply:
I was following the trend of the past wiki pages that includes the excerpt from the front page as the intro to the specific wiki page. I can change this if you think it would be better?

Reply:
oh... that makes sense
this is fine for now, (trying to be better about not letting the perfect be the enemy of the good,) but it'll probably be worth revisiting soon (both here and on the front page)